nais/kafka/aapen-k9statistikk-behandling-v2.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  name: aapen-k9statistikk-behandling-v2
+  namespace: k9saksbehandling
+  labels:
+    team: k9saksbehandling
+spec:
+  pool: {{kafka-pool}}
+  config:
+    cleanupPolicy: delete  # delete, compact
+    minimumInSyncReplicas: 1
+    partitions: 3
```

Comment:
Er det ok med tre partisjoner mot dvh, eller må de ha eventene i rekkefølge?

Reply:
Dette skal være ok, tror jeg. De får funksjonell tid på både Sak og Behandling, så de skal ikke trenge å få de i rekkefølge

Reply:
Georg legger inn behandlingUuid som nøkkel for å garantere rekkefølge.